src/components/Logout.js

```diff
@@ -1,10 +1,13 @@
-import { Link } from 'react-router-dom'
+import { Button } from '@chakra-ui/react'
+import useAuth from '../useAuth'
 
 const Logout = () => {
+    const removeAuth = useAuth().removeAuth;
+    
     return (
         <>
             <h1>Logout</h1>
-            <Link to="../">Login</Link>
+            <Button onClick={removeAuth}>Logout</Button>
```

Comment:
When I click it once - it takes me to `/logout` route and nothing happens. When I click it a second time - it logs me out.